### DISCUSSION TITLE
Flathub Upload

### DISCUSSION THREAD

**Discussion by jclds139** [saivert/pwvucontrol#14] [Ideas]
It looks like the [flathub submission process](https://docs.flathub.org/docs/for-app-authors/submission/) is pretty simple, as long as it builds locally in the first place. It sounds like they prefer the author to submit, so hopefully this can get started @saivert.
Any help you need to get that done?

**Comment by saivert** [comment]
Yes I need a bit of help. Since this is a rust project I need to vendor dependencies as the flatpak needs to be built in a sandboxed environment without any network access.

Also the linter claims I'm missing a screenshot in the metainfo file.

Also I probably need to coordinate these changes with a tagged release so that the git repo actually contains the up to date stuff when flatpak-builder checks out the tag.


[Current manifest which is flathub compatible](https://gist.github.com/saivert/98d9d2899a67b64dac12f550ed6b5ffb)

**Comment by saivert** [comment]
Currently in the process of submitting it to flathub now.
